specification/deviceprovisioningservices/Deviceprovisioningservices.Management/ProvisioningServiceDescription.tsp

```diff
@@ -0,0 +1,238 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+using Azure.ResourceManager.Private;
+namespace Microsoft.Devices;
+/**
+ * The description of the provisioning service.
+ */
+#suppress "@azure-tools/typespec-azure-core/no-private-usage" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@Azure.ResourceManager.Private.armResourceInternal(IotDpsPropertiesDescription)
```

Comment:
What is the need for using the private decorators vs using `is TrackedResource<>`

Reply:
Because of this issue. https://github.com/Azure/typespec-azure/issues/1979

But that issue will not be a problem later. We will remove these private decorators later.